workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/MonoBehaviourActivationManager.cs

```diff
@@ -7,9 +7,10 @@ namespace Improbable.Gdk.Core.GameObjectRepresentation
 {
     /// <summary>
     ///     Keeps track of Reader/Writer availability for SpatialOSBehaviours on a particular GameObject and decides when
-    ///     a SpatialOSBehaviour should be enabled, calling into the SpatialOSBehaviourLibrary for injection.
+    ///     a SpatialOSBehaviour should be enabled, calling into the SpatialOSBehaviourLibrary for injection, storing
+    ///     the created Readers/Writers in the given ReaderWriterStore.
     /// </summary>
-    internal class SpatialOSBehaviourManager
+    internal class MonoBehaviourActivationManager
```

Comment:
I think this name can be better, something in between `SpatialOSBehaviour` and `MonoBehaviour`. It deals with a special kind of `MonoBehaviour`s.

Reply:
In the MonoBehaviour design we agreed on SpatialOSBehaviour meaning a MonoBehaviour which [Require]s things, so that did make sense, but I'd suggest for changing it because 1) that meaning is not intuitive, 2) it's long, 3) all MonoBehaviours go through it, it just finds nothing to do on the ones without [Require] statements.

Reply:
Makes sense, resolving this comment